### PR TITLE
HX1D: updting linter errors and removing old  pytest markers

### PR DESCRIPTION
**Pull request by jghouse88**
## Changes proposed in this PR:
- Fixing linter errors
- Removing old pytest markers from `test_heat_exchanger_1D.py`

### Legal Acknowledgement

By contributing to this software project, I agree to the following terms and conditions for my contribution:

1. I agree my contributions are submitted under the license terms described in the LICENSE.txt file at the top level of this directory.
2. I represent I am authorized to make the contributions and grant the license. If my employer has rights to intellectual property that includes these contributions, I represent that I have received permission to make contributions and grant the required license on behalf of that employer.

### CODE REVIEW

File: idaes/generic_models/unit_models/heat_exchanger_1D.py [hunk 1]
```diff
@@ -1,15 +1,15 @@
...
 # by the software owners: The Regents of the University of California, through
 # Lawrence Berkeley National Laboratory,  National Technology & Engineering
-# Solutions of Sandia, LLC, Carnegie Mellon University, West Virginia University
-# Research Corporation, et al.  All rights reserved.
+# Solutions of Sandia, LLC, Carnegie Mellon University, West Virginia
+# University Research Corporation, et al.  All rights reserved.
 #
 # Please see the files COPYRIGHT.md and LICENSE.md for full copyright and
 # license information.
-#################################################################################
```
Comment (ksbeattie) [inline idaes/generic_models/unit_models/heat_exchanger_1D.py:12]: I think this might run afoul of the addheader check/test (if it exists)

Comment (jghouse88) [reply idaes/generic_models/unit_models/heat_exchanger_1D.py:12]: Is that why my tests are failing here?

**Review by eslickj — APPROVED** [review]
This looks good.

**Review by Andresj89 — APPROVED** [review]
LGTM

**Pull request merged**